documentation/docs/01-project-structure.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Project Structure
+---
+
+The basic structure of a SvelteKit project is:
+
+```sh
+my-app/
+ |
+ +---- .svelte-kit/                 # (generated at dev)
+ +---- build/                       # (generated at build)
+ +---- src/
+ |       |
+ |       +---- lib/                 # (optional)
+ |       +---- params/              # (optional)
+ |       +---- routes/
+ |       +---- app.css              # (optional)
+ |       +---- app.d.ts             # (optional)
+ |       +---- app.html
+ |       +---- hooks.js             # or 'hooks/' (optional)
+ |       +---- service-worker.js    # (optional)
+ +---- static/
+ +---- tests/                       # (optional)
+ +---- .eslintrtc.cjs               # (optional)
```

Comment:
this may change from `.cjs` to `.js`: https://github.com/sveltejs/kit/pull/5263

Reply:
it seems ESM for eslint doesn't work.